db/migrate/20191126181009_seed_test_results.rb

```diff
@@ -0,0 +1,63 @@
+class SeedTestResults < ActiveRecord::Migration[5.2]
+  def up
+    migrated = 0
+    ProfileHost.find_each do |profile_host|
+      host = profile_host.host
+      profile = profile_host.profile
+      test_result = TestResult.create(
+        host: host,
+        profile: profile
+      )
+
+      puts(" - Migrating results for host #{host.name}"\
+           " - profile #{profile.name}")
+      rule_results_by_start_time = scan_results(host, profile)
+        .group_by(&:start_time)
```

Comment:
This line here is key - I am grouping rule results by start time, host, profile so that we can infer "TestResults" from these. This allows us to migrate all rule results, not just the latest ones